backend/dal/Migrations/v01.09.02/Up/PostUp/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,4 @@
+PRINT 'Updating NotificationTemplates - Remove CC'
+
+UPDATE dbo.[NotificationTemplates]
+SET [Cc] = REPLACE([Cc], 'Deibert, Yvonne CITZ:EX <Yvonne.Deibert@gov.bc.ca>; Uppal, Sanjay CITZ:EX <Sanjay.Uppal@gov.bc.ca>; Miller, Lauren CITZ:EX <Lauren.Miller@gov.bc.ca>', '')
```

Comment:
Remove emails from CC.

Reply:
Hi Jeremy, it looks like the cc should be going to the RealPropertyDivision.Disposals@gov.bc.ca email address, but I'm assuming that this email address is being bcc'd because my local db appears to show that.